haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A help file viewer"
+DESCRIPTION="An application for viewing help files that use a simple html like mark-up. "
+
+HOMEPAGE="http://fatelk.com/"
+COPYRIGHT="2017-2019 Jim Saxton, Fat Elk Software"
+REVISION="1"
+LICENSE="FatElk"
+SOURCE_URI="http://fatelk.com/helpviewer/helpviewer-1.6.1.1.zip"
+CHECKSUM_SHA256="f780257f194cdf7c7ea2600cff8d82fd9d74681de80dd176e262aa36905133a9"
+
+SOURCE_DIR="helpviewer"
+
+ARCHITECTURES="x86_gcc2 !x86 x86_64"
+
+PROVIDES="
+	helpviewer = $portVersion
+	app:HelpViewer = $portVersion
+	cmd:helpviewer = $portVersion
+	"
+REQUIRES="
+	haiku
+	yab >= 1.7.5.5
+	lib:libyab
+	lib:libz
+	"
+
+BUILD_REQUIRES="
+	haiku_devel >= r1~alpha4_pm_hrev49976
+	"
+
+BUILD_PREREQUIRES="
+	cmd:unzip
+	"
+
+BUILD()
+{
+	unzip $targetArchitecture/HelpViewer.zip -d apps
+	unzip $targetArchitecture/mime_db.zip -d data
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	mkdir $appsDir/HelpViewer
+	mkdir -p $dataDir
+	mkdir -p $binDir
+	cp -r apps/* $appsDir/HelpViewer/
+	chmod a+x $appsDir/HelpViewer/HelpViewr-post-install.sh
+	cp -r data/mime_db $dataDir/
+	ln -s $appsDir/HelpViewer/HelpViewer $binDir
```

Comment:
Maybe add a Deskbar link here with addAppDeskbarSymlink

Reply:
This is done intentionally. Launching the program by its self is only needed for developers. The prlgram should be launched from a control in the program that HelpViewer is used to document.

Reply:
Thanks for clearing this up :+1: